### PR TITLE
Gpg/homepage2021

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

Add new homepage
- cleanup old landing pages and assets
- Dynamic data from results (with fallback) 
- Background gradient animations
- New footer
##### Testing

localhost_8000_

### CODE REVIEW

File: app/app/urls.py [hunk 1]
```diff
@@ -487,7 +487,7 @@
     re_path(r'^results/?', retail.views.results, name='results'),
     re_path(r'^activity/?', retail.views.activity, name='activity'),
     re_path(r'^townsquare/?', townsquare.views.town_square, name='townsquare'),
-    re_path(r'^$', townsquare.views.landing_toggle, name='index'),
```
Comment (thelostone-mc) [inline app/app/urls.py:490]: can we remove `landing_toggle` itself cause i'm assuming it's dead code

Comment (octavioamu) [reply app/app/urls.py:490]: let me check what is that one for

Comment (octavioamu) [reply app/app/urls.py:490]: donee

**Review by thelostone-mc — APPROVED** [review]

**Review by chibie — APPROVED** [review]
smooth!!

File: app/retail/views.py [hunk 2]
```diff
@@ -78,8 +78,26 @@ def get_activities(tech_stack=None, num_activities=15):
...
+            'universe_total_usd': data['universe_total_usd'] if data['universe_total_usd'] else 0,
+            'human_universe_total_usd': f"${round(data['universe_total_usd'] / 1000000, 1)}m" if data['universe_total_usd'] else 0,
+            'mau': data['mau'] if data['mau'] else 0
+        }
+    except:
+        data_results = {
+            'universe_total_usd': 18874053.680999957,
+            'human_universe_total_usd': "$18.9m",
+            'mau': 161205.0
+        }
```
Comment (gdixon) [inline app/retail/views.py:97]: Nice!! 🚀

File: app/retail/templates/shared/footer.html [hunk 3]
```diff
@@ -17,84 +17,98 @@
...
-            </a>
-            <a href="https://gitcoin.co/wiki/wiki-code-conduct/">{% trans "Code of Conduct" %}</a>
-            <a href="{% url "request_money" %}">{% trans "Request Money" %}</a>
-            <a href="/wiki/">{% trans "Wiki" %}</a>
-            <a href="https://github.com/gitcoinco/web/issues/new/choose">{% trans "Help" %}</a>
-            <a href="https://store.gitcoin.co/">{% trans "Store" %}</a>
+            <a class="link-current hover-underline mb-2" target="_blank" href="https://discord.gg/jWUzf7b8Yr">Discord</a>
+            <a class="link-current hover-underline mb-2" href="https://gitcoin.co/wiki/wiki-code-conduct/">{% trans "Code of Conduct" %}</a>
+            <a class="link-current hover-underline mb-2" href="{% url "request_money" %}">{% trans "Request Money" %}</a>
+            <a class="link-current hover-underline mb-2" href="/wiki/">{% trans "Wiki" %}</a>
```
Comment (gdixon) [inline app/retail/templates/shared/footer.html:99]: Consider: Should we update these to use django URL tags?

File: app/assets/v2/scss/gitcoin.scss [hunk 4]
```diff
@@ -203,6 +205,10 @@
   background-color: #F5F2FF;
 }
 
+.bg-violet-300 {
+  background-color: $gc-violet-300;
+}
+
```
Comment (gdixon) [inline app/assets/v2/scss/gitcoin.scss:211]: Consider: Should we move all of these color/font utilities from `gitcoin.scss` to `gc-utilities.scss`?

**Review by gdixon — APPROVED** [review]
Awesome work @octavioamu, @PixelantDesign and @willsputra!!

Code and design are looking great!! Let's get this merged!! 🚀

**Pull request merged**